test/edit-column.html

```diff
@@ -544,6 +544,25 @@
         });
       });
 
+      describe('disable editing', () => {
+
+        let grid, input;
+
+        beforeEach(() => {
+          grid = fixture('default');
+          grid.items = getItems();
+          flushGrid(grid);
+        });
+
+        it('should not show editor when editing is disabled', () => {
+          const cell = getContainerCell(grid.$.items, 1, 0);
+          grid._disableEditing = true;
+          enter(cell);
```

Comment:
Only the "keydown" case is tested even though changes were made to "click" and "dblclick" also. Either add tests for all the cases or do the `if (this._disableEditing) { return }` check in one central place (`_enterEditFromEvent` maybe?)

Reply:
Done.